src/scss/patterns/_sizes.scss

```diff
@@ -16,22 +12,22 @@
 
 // Extend thing outside its container on one side only
 .extend-left {
-  @include extend-side('left');
+  --column: wide / page;
```

Comment:
I don't think I've seen this `/` syntax used anywhere. Or is this literally dividing these two values? lol

Reply:
This is being used in the `grid-column` property, where `/` separates `grid-column-start` and `grid-column-end`. Combined with area/line names, `wide / page` starts at the `wide-start` line, and ends at the `page-end` line.

Reply:
oh, yeah, ha ha. That makes sense.